src/Receipt/DonationDetailsGroup/Details/Amount.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Give\Receipt\DonationDetailsGroup\Details;
+
+use Give\Receipt\Detail;
+use function give_get_payment_meta as getDonationMetaData;
+use function give_format_amount as formatAmount;
+use function give_currency_filter as filterCurrency;
+
+/**
+ * Class Amount
+ *
+ * @since 2.7.0
+ * @package Give\Receipt\DonationDetailsGroup\Details
+ */
+class Amount extends Detail {
+	/**
+	 * @inheritDoc
+	 */
+	public function getLabel() {
+		return __( 'DONATION AMOUNT', 'give' );
```

Comment:
For this and the other labels, they should be localized as sentence case, and then changed to all caps via PHP.

Reply:
@bradp no need to add logic to convert string to upper case. Let developers decide how they want their string to render on a string.